### PR TITLE
ConfigurationGathering.py: Refactor targets list

### PR DESCRIPTION
**Pull request by ujjaldas1997**
Refactoring of ``targets`` list ensures
simplified one line list comprehension
of ``targets`` list

Closes https://github.com/coala/coala/issues/3708

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/settings/ConfigurationGathering.py [hunk 1]
```diff
@@ -241,8 +241,8 @@ def load_configuration(arg_list,
 
     targets = []
```
Comment (RaiVaibhav) [inline coalib/settings/ConfigurationGathering.py]: No need of defining `targets` here

**Comment by ujjaldas1997** [comment]
Changed. Please see the updated commit

**Comment by RaiVaibhav** [comment]
ack 7723b0b29022e53546d4b23a79a7611e82bb6ae5

**Comment by RaiVaibhav** [comment]
Why did you close the PR ??

**Comment by ujjaldas1997** [comment]
@RaiVaibhav  I have closed it mistakenly. Is there any possibility of reopening it?

**Comment by RaiVaibhav** [comment]
Where is your commit?, You deleted the commit or branch something??

**Comment by ujjaldas1997** [comment]
No, I just rebased it. But I had reset HEAD to the commit where I had rebased. But still the commit is not restored.

**Comment by RaiVaibhav** [comment]
Please read git basics : disappointed:

**Comment by ujjaldas1997** [comment]
Will I pm you on gitter?

**Comment by RaiVaibhav** [comment]
My train is in 15 min and after that you about Indian railways and about their net connectivity

I will suggest you to read first apply it on your personal repo, understand it after that you will not need any help :wink:

**Pull request closed without merging**